bindings/nodejs/examples/exchange/1-create-account.ts

```diff
@@ -12,56 +12,48 @@ require('dotenv').config({ path: '.env' });
 
 async function run() {
     try {
-        if (!process.env.WALLET_DB_PATH) {
+            for (const envVar of ['WALLET_DB_PATH','NODE_URL','STRONGHOLD_SNAPSHOT_PATH','STRONGHOLD_PASSWORD','MNEMONIC'])
+        if (!(envVar in process.env)) {
             throw new Error(
-                '.env WALLET_DB_PATH is undefined, see .env.example',
+                `.env ${envVar} is undefined, see .env.example`,
             );
         }
-        if (!process.env.NODE_URL) {
-            throw new Error('.env NODE_URL is undefined, see .env.example');
-        }
-        if (!process.env.STRONGHOLD_SNAPSHOT_PATH) {
-            throw new Error(
-                '.env STRONGHOLD_SNAPSHOT_PATH is undefined, see .env.example',
-            );
-        }
-        if (!process.env.STRONGHOLD_PASSWORD) {
-            throw new Error(
-                '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
-            );
-        }
-        if (!process.env.MNEMONIC) {
-            throw new Error('.env MNEMONIC is undefined, see .env.example');
-        }
-
-        const walletOptions: WalletOptions = {
-            storagePath: process.env.WALLET_DB_PATH,
-            clientOptions: {
-                nodes: [process.env.NODE_URL],
-            },
-            coinType: CoinType.IOTA,
-            secretManager: {
-                stronghold: {
-                    snapshotPath: process.env.STRONGHOLD_SNAPSHOT_PATH,
-                    password: process.env.STRONGHOLD_PASSWORD,
+       
+        if (process.env.NODE_URL&&process.env.STRONGHOLD_SNAPSHOT_PATH&&process.env.STRONGHOLD_PASSWORD){
+            const walletOptions: WalletOptions = {
+                storagePath: process.env.WALLET_DB_PATH,
+                clientOptions: {
+                    nodes: [process.env.NODE_URL],
                 },
-            },
-        };
+                coinType: CoinType.IOTA,
+                secretManager: {
+                    stronghold: {
+                        snapshotPath: process.env.STRONGHOLD_SNAPSHOT_PATH,
+                        password: process.env.STRONGHOLD_PASSWORD,
+                    },
+                },
+            };
+        
+       
 
         const wallet = new Wallet(walletOptions);
 
         // Mnemonic only needs to be set the first time.
+        if(process.env.MNEMONIC){
         await wallet.storeMnemonic(process.env.MNEMONIC);
 
+        }
```

Comment:
This extra if everywhere is very annoying, can't we just do this?
```suggestion
        await wallet.storeMnemonic(process.env.MNEMONIC as string);
```

Reply:
Btw to format the examples you can use `yarn run format`

Reply:
Thanks for the correction, I'll do that and I checked the yarn run format later after seeing the test failure.